Ginger/GingerCoreNET/Run/RunListenerLib/ExecutionLogger.cs

```diff
@@ -241,7 +241,7 @@ public virtual void SetReportRunSet(RunSetReport runSetReport, string logFolder)
         public abstract string GetLogFolder(string folder);
 
 
-        public abstract Task<bool> SendExecutionLogToCentralDBAsync(LiteDB.ObjectId runsetId, Guid executionId, eDeleteLocalDataOnPublish deleteLocalData);
+        public abstract Task<bool> SendExecutionLogToCentralDBAsync(LiteDB.ObjectId runsetId, Guid executionId, eDeleteLocalDataOnPublish deleteLocalData, Guid runsetGuid);
```

Comment:
i don't think we need to pass the Runset GUID for all those methods, we can take it from Workspace.RunsetExecuter.Runset